src/Components/CreateImageWizard/steps/FileSystem/components/AdvancedPartitioning.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import {
+  Alert,
+  Button,
+  CodeBlock,
+  Content,
+  ContentVariants,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import { useAppSelector } from '../../../../../store/hooks';
+import {
+  selectDiskMinsize,
+  selectDiskPartitions,
+  selectDiskType,
+} from '../../../../../store/wizardSlice';
+
+const AdvancedPartitioning = () => {
+  const minsize = useAppSelector(selectDiskMinsize);
+  const type = useAppSelector(selectDiskType);
+  const diskPartitions = useAppSelector(selectDiskPartitions);
+
+  return (
+    <>
+      <Content>
+        <Content component={ContentVariants.h3}>Configure disk layout</Content>
+      </Content>
+      <Alert
+        variant='info'
+        title='Disk can be currently only edited directly in the blueprint'
+      />
+      <Content>
+        <Content>Define complete partition table for your image.</Content>
+        <Content>
+          The order of partitions may change when the image is installed in
+          order to conform to best practices and ensure functionality.
+          <br></br>
+          <Button
+            variant='link'
+            icon={<ExternalLinkAltIcon />}
+            iconPosition='right'
+            className='pf-v6-u-pl-0'
+          >
+            Read more about advanced partitioning here
```

Comment:
Also the button currently doesn't lead anywhere - tried to search through the docs, not sure if we have a good place to link though